setup.sh

```diff
@@ -4,3 +4,4 @@ sudo make install
 ruby-install ruby
 ruby-install jruby
 ruby-install rubinius
+ruby-install truffleruby
```

Comment:
I was not sure if I should add it here.

Reply:
I think it does make sense to add TruffleRuby here, assuming the point is to install the Ruby platforms that are commonly benchmarked these days. This file wasn't updated when the `--latest` flag was added, so that would probably also make sense to add across the board.

That said, I wonder if it'd be better to deprecate the feature and remove this file. @postmodern, should `setup.sh` still be a thing? If yes, should `--latest` be added?